src/node_options.cc

```diff
@@ -27,7 +27,7 @@ std::shared_ptr<PerProcessOptions> cli_options{new PerProcessOptions()};
 }  // namespace per_process
 
 void DebugOptions::CheckOptions(std::vector<std::string>* errors) {
-#if !NODE_USE_V8_PLATFORM
+#if !NODE_USE_V8_PLATFORM && !HAVE_INSPECTOR
   if (inspector_enabled) {
     errors->push_back("Inspector is not available when Node is compiled "
                       "--without-v8-platform");
```

Comment:
probably the error message also be modified as `--without-v8-platform and --without-inspector` ?

Reply:
ah good catch - i'll update this! you're right, that's more clear for users.